lavague-core/lavague/core/evaluator.py

```diff
@@ -118,18 +116,16 @@ def evaluate(
         try:
             for i, row in tqdm(results.iterrows()):
                 driver.__init__()  # reinit the driver
-                action = parse_action(row["action"])
-                if driver:  # artificially get the page if the retriever needs a driver
```

Comment:
Is this condition relevant? The code would crash at line 118 if the driver was `None`

Reply:
well spotted, I added the init afterwards and forgot about it.
To answer the question: for now the driver always exists, but in theory, it's possible to do retrieval without a driver (it's a niche case though)

Reply:
fixed it